_examples/cluster-broadcast/node1/main.go

```diff
@@ -2,28 +2,37 @@ package main
 
 import (
 	"fmt"
-	"github.com/AsynkronIT/goconsole"
+	"time"
+
+	console "github.com/AsynkronIT/goconsole"
+	"github.com/AsynkronIT/protoactor-go/_examples/cluster-broadcast/shared"
```

Comment:
@otherview  Hello. :smiley: 
This may not work with `go module`, but `GOPATH` with `vendor`.
Here is my pr for  `go module` #395 , but it would breaking `GOPATH` with `vendor`.